commit 91ac3b0ac9f9579aeb88f9f9d0a2ff9814e3cbe8 (refs/changes/61/261/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2016-08-09T15:07:50-07:00 (3 years, 2 months ago)
    
    DOCKER-908: cn-agent - remove test files from npm install of sdc-docker-build

diff --git a/.npmignore b/.npmignore
new file mode 100644
index 0000000..e757aa0
--- /dev/null
+++ b/.npmignore
@@ -0,0 +1,6 @@
+/.npmignore
+/deps
+/node_modules
+/test
+/tools
+/Makefile
